### DISCUSSION TITLE
Semantic relation path for reaching the leaf node with contains LDES Members 

### DISCUSSION THREAD

**Discussion by xdxxxdx** [Informatievlaanderen/VSDS-LDESServer4J#920] [General]
LDES Server version: ldes/ldes-server:2.0.1-SNAPSHOT

This is a follow-up case for the discussion on whether should we have lateral relations on SLACK: https://vsdscore.slack.com/archives/C03DHRBGDQ9/p1695036769839969

Testbed is made for checking if the member in the `tree:Node `linked in a `tree:Relation` (with its subclasses) conforms the relation statement.
e.g. for Geolocation: e.g.https://github.com/Informatievlaanderen/VSDS-Testbed/blob/main/test-services/src/main/java/be/vlaanderen/ldes/handlers/RelationGeospatialValidationHandler.java

In our current implementation: e.g `tree:GeospatialFragmentation`. A tree structure is generated as:
![image](https://github.com/Informatievlaanderen/VSDS-LDESServer4J/assets/15314807/181639dc-64a5-4f43-827e-8c19900c2997)

To arrive at the` tree:Node ` which contains `tree:members` (in current case： http://[localhost:8080/kbo/by-location?tile=15/16899/11116&pageNumber=1](http://localhost:8080/kbo/by-location?tile=15/16899/11116&pageNumber=1)).
Needs to go through generic relation: `tree:Relation `.
This path requires saving the status of the compared value of the `tree:member ` back to the last Semantic Relation (tree:GeospatiallyContainsRelation), In this current case where 1 marked

My idea here is, that I feel it is important to have a semantic meaning for relations to reach the Node which contains the LDES members. 
Like this, it is easier to know which values should be contained inside the member. 
Basically something like this:
![image](https://github.com/Informatievlaanderen/VSDS-LDESServer4J/assets/15314807/671a6da6-7536-4bbb-bb73-a85172a982d9)

Thanks
Please feel free to leave comment
Miro: https://miro.com/app/board/uXjVMiATqP8=/?share_link_id=273030385087
